doc/articles/getting-started-tutorial-1.md

```diff
@@ -0,0 +1,412 @@
+---
+title: Tutorial - Creating a Single Page App with Uno
+description: How to get started developing Cross Platform apps with Uno
+---
+
+# Create a Single Page App with Uno
+
+[Download Sample](https://github.com/nventive/Uno.GettingStarted)
+
+In this tutorial you will learn how to:
+
+- Add the Uno Project Templates to Visual Studio
+- Create a new Project with Uno
+- Learn basics on Model Binding
+
+The tutorial walks through creating a cross platform application with Uno, which enables you to see a single Issue entry.
+
+### Prerequisites
+
+- Visual Studio 2019 (latest release), with the **Mobile development with .NET** workload installed.
```

Comment:
We could add more details on all the checkboxes required in the Visual Studio Installer, we also need some web development ones I think, we also suggest specific UWP windows toolkit

Reply:
fixed